config/config.go

```diff
@@ -67,6 +69,8 @@ type terragruntConfigFile struct {
 	TerragruntDependencies      []Dependency              `hcl:"dependency,block"`
 	GenerateBlocks              []terragruntGenerateBlock `hcl:"generate,block"`
 	RetryableErrors             []string                  `hcl:"retryable_errors,optional"`
+	RetryMaxAttempts            *int                      `hcl:"retry_max_attempts,optional"`
+	RetrySleepInterval          *int                      `hcl:"retry_sleep_interval,optional"`
```

Comment:
NIT: should it be `retry_sleep_interval_sec` to make the unit clear from the name?